src/Twig/Extension/PageExtension.php

```diff
@@ -218,4 +221,30 @@ public function controller(string $controller, array $attributes = [], array $qu
 
         return HttpKernelExtension::controller($controller, $attributes, $query);
     }
+    
+    /**
+     * @param PageInterface $page
+     * @param array $parameters
+     * @return string
+     */
+    public function pageUrl(PageInterface $page, array $parameters = []) : string
```

Comment:
IMO you should create test for those 2 new methods!

Reply:
So what should i do ? Use the`cmsPageRouter as @VincentLanglet  proposed or keep the cmf_routing_object and create tests ?

Reply:
I'm not saying to use hardcoded string `cmf_routing_object`!
and any solution that you choose you should write test!

Reply:
Ok, i'll do my best to answer your demand, when i have some free time I go back on the subject